libs/sdk-backend-tiger/src/backend/workspace/attributes/elements/tests/date.test.ts

```diff
@@ -0,0 +1,29 @@
+// (C) 2021 GoodData Corporation
+
+import {
+    newRelativeDateFilter,
+    IRelativeDateFilter,
+    idRef,
+    DateAttributeGranularity,
+} from "@gooddata/sdk-model";
+
+import { getRelativeDateFilterShiftedValues } from "../date";
+
+const generateRelativeDateFilter = (granularity: DateAttributeGranularity) =>
+    newRelativeDateFilter(idRef("gdc/uri/rf"), granularity, -3, 0);
```

Comment:
Nitpick: for idRefs, you can skip the idRef call in these factories. So the following should be equivalent:
```ts
newRelativeDateFilter("gdc/uri/rf", granularity, -3, 0);
```

Reply:
fixed in https://github.com/gooddata/gooddata-ui-sdk/pull/1678/commits/0aaa64dd197b99e232ef810d0783ef604638bb5c